GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -251,6 +259,24 @@ + (void)load {
   [self sharedApplication];
 }
 
+/** Initializes sSemaphoreQueue and sBackgroundIdentifierToSemaphoreMap. */
```

Comment:
a docstring (comments that look like `/** */`) aren't necessary here, +initialize is a method whose declaration exists elsewhere. You can just use an implementation comment (`//`)

Reply:
remove comment for initialize